src/pocketmine/block/Prismarine.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____  
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \ 
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/ 
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_| 
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ * 
+ *
+*/
+
+namespace pocketmine\block;
+
+use pocketmine\item\Item;
+use pocketmine\item\Tool;
+use pocketmine\item\enchantment\enchantment;
+
+class Prismarine extends Solid{
+	const NORMAL = 0;
+	const DARK = 1;
+	const BRICKS = 2;
+
+	protected $id = self::PRISMARINE;
+
+	public function __construct($meta = 0){
+		$this->meta = $meta;
+	}
+
+	public function getHardness() {
+		return 1.5;
+	}
+
+	public function getToolType(){
+		return Tool::TYPE_PICKAXE;
+	}
+
+	public function getName() : string{
+		static $names = [
+			self::NORMAL => "Prismarine",
+			self::DARK => "Dark Prismarine",
+			self::BRICKS => "Prismarine Bricks"
+		];
+		return $names[$this->meta & 0x07];
+	}
+
+	public function getDrops(Item $item) : array {
+		return [
+			[Item::PRISMARINE, $this->getDamage(), 1],
```

Comment:
@KairusDarkSeeker Don't give up on it, just fix it, all the issue is, is the way it's formatted.